File: c-language/calculator.c
//This is my first project in c-language 
//This is simple and basic calculator to perform basic math tasks
//Here is the source code the my project 
//Support me develop this paroject futher and gain valuable knowledge regarding coding in c-language
//I hope you will enjoy this program........👍👍👍👍

#include<stdio.h>
int a,b;
char c;
int main(){
    printf("This is a simple calculator!!!!");
    printf("\nCreated in 'C' language");
    printf("/nCreated by D.varma");
    printf("\nLet's go ..............");
    gather_information();
    Get_operator();
    Condition();
}
void Condition(){
    if(c =='+'){
        printf("\nThen the result is : %d",a+b);
        return_the_function();
    }
    else if(c =='-'){
        printf("\nThen the result is : %d",a-b);
        return_the_function();
    }
    else if(c =='*'){
        printf("\nThen the result is : %d",a*b);
        return_the_function();
    }
    else if(c == '/'){
        printf("\nThen the result is : %d",a/b);
        return_the_function();
    }
    else{
        printf("Please Enter the correct operator....");
        Get_operator();
        Condition();
    }
}
void Get_operator(){
    printf("\nEnter the operator:");
    scanf("%s",&c);
}
void gather_information(){
    printf("\nEnter the first number:");
    scanf("%d",&a);
    printf("Enter the second number:");
    scanf("%d",&b);
    printf("Here are the few operator's to perform the maths ");
    printf("( '+' , '-' , '*' , '/' )");
}
void return_the_function(){
    printf("\n\n*****Use it one more time*****");
    gather_information();
    Get_operator();
    Condition();
}

File: python-language/calculator.py
#This is my first project in python-language 🥇🥇🥇
#This is simple and basic calculator to perform basic math tasks
#Here is the source code the my project 
#Support me develop this paroject futher and gain valuable knowledge regarding coding in python-language
#I hope you will enjoy this program........👍👍👍👍


#bin/usr/env python3
num1 = int(input("Enter the fitst number : "))
num2 = int(input("Enter the second number : "))
reponse = str(input("To know about the operators type 'y/n' :"))
if reponse =='y':
    print("Here are the few operators to perform the math task!")
    print(" '+' = Addition\n '-' = subraction\n '/' = division \n '*' = multipluction\n '**' = expontental")
    operator = input("Enter the operator :")
elif reponse=='n':
    operator = input("Enter the operator :")
else:
    print("Please enter correct input.....")
if operator=='+':
    print("The answer is : ",num1+num2)
elif operator=='-':
    print("The answer is : ",num1-num2)
elif operator=='*':
    print("The answer is : ",num1*num2)
elif operator =='/':
    print("The answer is :",num1/num2)
elif operator == '**':
    print("The answer is : ",num1**num2)
else:
    print("Please Enter correct operator.....")
exit()
